integration/simulation/network/inmemory.go

```diff
@@ -37,9 +37,9 @@ func (n *basicNetworkOfInMemoryNodes) Create(params *params.SimParams, stats *st
 
 	// Invent some addresses to assign as the L1 erc20 contracts
 	dummyOBXAddress := datagenerator.RandomAddress()
-	params.Wallets.Tokens[bridge.HOC].L1ContractAddress = &dummyOBXAddress
+	params.Wallets.Tokens[rollupextractor.HOC].L1ContractAddress = &dummyOBXAddress
```

Comment:
do we still need he hoc and poc hardcoded in the main codebase?

Reply:
ok, pls add a todo to get rid of them ASAP.
Let's remove all trace of them from the codebase when we can

Reply:
https://github.com/obscuronet/obscuro-internal/issues/1158 - i've updated this ticket